### PR TITLE
Show header depending on test run status

### PR DESCRIPTION
**Pull request by kntsoriano**

### CODE REVIEW

File: app/javascript/components/editor/TestRunSummary.tsx [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+  switch (testRun.status) {
+    case TestRunStatus.FAIL:
+      return (
+        <div className="summary-status failed">
+          <span className="--dot" />1 test failure
+        </div>
+      )
+    case TestRunStatus.PASS:
+      return (
+        <div className="summary-status">
```
Comment (kntsoriano) [inline app/javascript/components/editor/TestRunSummary.tsx]: @iHiD I leave this to you to put the correct CSS class.

**Comment by iHiD** [comment]
Added a couple of commits 🙂 

Also added a couple of TODOs to the OP of the main editor PR based on stuff I noticed.

File: Procfile.dev [hunk 2]
```diff
@@ -1,4 +1,4 @@
 server: bin/rails server -p 3020
 assets: bin/webpack-dev-server
 anycable: RAILS_ENV=development bundle exec anycable --server-command "anycable-go --host=localhost --port 3334"
-js-tests: yarn test-watch
+#js-tests: yarn test-watch
```
Comment (ErikSchierboom) [inline Procfile.dev]: I'm fine with just completely removing it :) I have a dislike of commented out code ;)

File: app/javascript/components/editor/TestRunSummary.tsx [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
 import { TestRunChannel } from '../../channels/testRunChannel'
 import { TestRunSummaryContent } from './TestRunSummaryContent'
 import { fetchJSON } from '../../utils/fetch-json'
 
+const TestRunSummaryHeader = ({ testRun }: { testRun: TestRun }) => {
+  switch (testRun.status) {
+    case TestRunStatus.FAIL:
+      return (
+        <div className="summary-status failed">
+          <span className="--dot" />1 test failure
```
Comment (ErikSchierboom) [inline app/javascript/components/editor/TestRunSummary.tsx:12]: In theory, there could be multiple test failures. Do we always want to list it as one failure?

**Review by ErikSchierboom — APPROVED** [review]

File: app/javascript/components/editor/TestRunSummary.tsx (hunk 3 above)
Comment (iHiD) [reply app/javascript/components/editor/TestRunSummary.tsx:12]: We show all the rest as skipped.

Comment (ErikSchierboom) [reply app/javascript/components/editor/TestRunSummary.tsx:12]: Okay!

File: Procfile.dev (hunk 2 above)
Comment (iHiD) [reply Procfile.dev]: Want to put a PR up that removes it, and ensures that README has instructions on how to run the JS tests?

**Pull request merged**